app/helpers/ndr_ui/bootstrap_helper.rb

```diff
@@ -243,6 +243,34 @@ def bootstrap_form_for(record_or_name_or_array, *args, &proc)
       end
     end
 
+    # Identical signature to form_with, but uses NdrUi::BootstrapBuilder.
+    # See ActionView::Helpers::FormHelper for details
+    def bootstrap_form_with(*args, &proc)
+      options = args.extract_options!
+      options[:html] ||= {}
+      horizontal = options.delete(:horizontal)
+
+      # :horizontal
+      if horizontal
+        # set the form html class for horizontal bootstrap forms
+        options[:html][:class] ||= ''
+        classes = (options[:html][:class].split(' ') << 'form-horizontal').uniq.join(' ')
+        options[:html][:class] = classes
+      end
+
+      # We switch autocomplete off by default
+      raise 'autocomplete should be defined an html option' if options[:autocomplete]
+      options[:html][:autocomplete] ||= 'off'
+
+      form_with(*(args << options.merge(builder: NdrUi::BootstrapBuilder))) do |form|
+        # Put the form builder into horizontal mode (if necessary)
+        form.horizontal_mode = horizontal if horizontal
+
+        # yield to the provided form block
+        yield(form)
```

Comment:
If this is `block.call(form)`, you shouldn't get the warnings about unused variables.